app/pages/system/silos/SilosPage.tsx

```diff
@@ -112,6 +114,12 @@ export function SilosPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Cloud24Icon />}>Silos</PageTitle>
+        <DocsPopover
+          heading="Silos"
+          icon={<Cloud16Icon />}
+          summary="Silos are collections of resources, designed to provide strict tenancy separation. They have separate UI and API endpoints, separate resource limits, and separate access policies."
```

Comment:
This is almost there, but I feel like the "collections of resources" bit hurts more than it helps. And "separate UI and API endpoints" could be phrased more concretely. Something like:

> Silos provide strict tenancy separation between groups of users. Each silo has its own resource limits and access policies as well as its own subdomain for the web console and API.